core/src/main/java/bio/terra/pearl/core/service/participant/ProfileService.java

```diff
@@ -45,6 +44,15 @@ public Map<UUID, Profile> loadAllWithMailingAddress(List<UUID> profileIds) {
                 .collect(Collectors.toMap(Profile::getId, Function.identity()));
     }
 
+    public List<Profile> loadAllByEnrolleeIdsWithMailingList(List<UUID> enrolleeIds) {
```

Comment:
should this be called `loadAllByEnrolleeIdsWithMailingAddress`?

Reply:
🤦  nice catch